packages/admin-ui/src/text-area/text-area-container.ts

```diff
@@ -0,0 +1,28 @@
+import { createComponent, useElement } from '@vtex/admin-ui-react'
+import type { VariantProps } from '@vtex/admin-ui-core'
+
+import * as inputStyle from '../text-input/text-input.style'
```

Comment:
I think is better for your style to import this and add to the `style.container` prop